### PR TITLE
Bump version to v0.4.4

### PR DESCRIPTION
**Pull request by AlexWaygood**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
...
+- Editor settings are used by default if no file-based configuration exists ([#11266](https://github.com/astral-sh/ruff/pull/11266))
+
+### Bug fixes
+
+- \[`pylint`\] Consider `with` statements for `too-many-branches` (`PLR0912`) ([#11321](https://github.com/astral-sh/ruff/pull/11321))
+- \[`flake8-blind-except`, `tryceratops`\] Respect logged and re-raised expressions in nested statements (`BLE001`, `TRY201`) ([#11301](https://github.com/astral-sh/ruff/pull/11301))
+
+### Other changes
+
+- Recognise assignments such as `__all__ = builtins.list(["foo", "bar"])` as valid `__all__` definitions ([#11335](https://github.com/astral-sh/ruff/pull/11335))
```
Comment (charliermarsh) [inline CHANGELOG.md:30]: Maybe just put under `Bug fixes` to remove this lonely section?

**Review by charliermarsh — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (zanieb) [reply CHANGELOG.md:30]: Yeah we basically always remove this section entirely.

Comment (AlexWaygood) [reply CHANGELOG.md:30]: Is this change a bugfix or a feature, I wonder 🤔 but yeah, I'll move it!

Comment (charliermarsh) [reply CHANGELOG.md:30]: Haha yeah, not sure -- I guess I'd say bugfix. Not a big deal either way, just simpler to move it elsewhere.

Comment (zanieb) [reply CHANGELOG.md:30]: Usually we'll call things like this a bug fix although they could be construed to be an enhancement 🤷‍♀️

**Pull request merged**